doc/release-notes/NEXT.md

```diff
@@ -4,3 +4,13 @@
   as it will lose any conflict resolution of the merge commit when it is rebased. 
   This is independent on the used commandline switch. (#1342)
 * Improve performance by caching branch objects instead of looking them up over and over (#1286)
+* Pin paket version to 5.251.0 to avoid spurious changes during build in paket generated files
```

Comment:
The next.md is used to do a user friendly release note.
So, could you keep just one sentence like "Upgrade to .NET Framework 4.7.2 & Upgrade dependencies (#1344 by @siprbaum)" and we're good to go.. .

Reply:
Done.